broker/src/main/java/io/zeebe/broker/transport/backpressure/CommandRateLimiter.java

```diff
@@ -65,7 +69,13 @@ public boolean tryAcquire(final int streamId, final long requestId, final Intent
   public void onResponse(final int streamId, final long requestId) {
     final Listener listener = responseListeners.remove(new ListenerId(streamId, requestId));
     if (listener != null) {
-      listener.onSuccess();
+      try {
+        listener.onSuccess();
+      } catch (final IllegalArgumentException e) {
+        LOG.trace("Ignoring request in command rate limiter due to: ", e);
```

Comment:
This should never happen because we're passing a custom clock but, if this happens at all then it would happen very often. The log would be filled with warning messages. Also, in that scenario, the consequence would be that the backpressure wouldn't be accurate but if the system clock is not accurate then there's not much we can do right? Maybe then debug level at most would be the appropriate, wdyt?

Reply:
On second thought, I changed them to warning in the latest commit as you requested